SwiftWeather/ForecastDateTime.swift

```diff
@@ -6,16 +6,19 @@
 import Foundation
 
 struct ForecastDateTime {
-  let rawDate: Double
-
-  init(_ date: Double) {
-    rawDate = date
-  }
-
-  var shortTime: String {
-    let dateFormatter = DateFormatter()
-    dateFormatter.dateFormat = "HH:mm"
-    let date = Date(timeIntervalSince1970: rawDate)
-    return dateFormatter.string(from: date)
-  }
+    let rawDate: Double
+    let timeZone: TimeZone
+    
+    init(date: Double, timeZone: TimeZone) {
```

Comment:
Sometimes, we will apply a default value for it like `init(date: Double, timeZone: TimeZone = TimeZone.current)`, then the app doesn't need to pass it and we can inject in unit tests.